### PR TITLE
BZ1694774: Included module to define a cluster admin.

### PR DESCRIPTION
**Pull request by huffmanca**
Included a module in the RBAC section that includes an example of defining a cluster admin.

This is for OS 4.x.

### CODE REVIEW

File: modules/rbac-creating-cluster-admin.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+
+ifdef::openshift-enterprise,openshift-origin[]
```
Comment (bergerhoffer) [inline modules/rbac-creating-cluster-admin.adoc]: Why is this line here? There's not a matching ending one. And if it is required, I'd think we'd want to put the conditional in the assembly? wdyt?

File: modules/rbac-creating-cluster-admin.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+[id="creating-cluster-admin-{context}"]
+= Example of creating a cluster admin
```
Comment (bergerhoffer) [inline modules/rbac-creating-cluster-admin.adoc]: Is there a reason that this title starts with "Example"? Could it just be "Creating a cluster admin"?

Comment (huffmanca) [reply modules/rbac-creating-cluster-admin.adoc]: You're right - this flows better as starting with `Creating`. Incorporated this change.

File: modules/rbac-creating-cluster-admin.adoc (hunk 1 above)
Comment (huffmanca) [reply modules/rbac-creating-cluster-admin.adoc]: It was an oversight from the base file that I used. It seems that conditionals are split between modules and assemblies, and one of the modules I started this from included it.

I've removed the conditional from the module. The other sections on defining user- and cluster- roles don't have a similar conditional, so I believe it's safe to keep out of the assembly.
